### ISSUE TITLE
Users should have their own set of public and private keys

### ISSUE THREAD

**Issue by MD-AZMAL** [Abhishek8218/file-sharing#2]
It appears that user's private and public keys are not generated. Please have a look at this

**Comment by Abhishek8218** [comment]
> It appears that user's private and public keys are not generated. Please have a look at this


Thanks for your review. In the code, I've used passphrase-based encryption, which doesn't require keys.  However, if key-based encryption is compulsory , I'm open to modify it accordingly. 

Let me know if the modification is needed.

**Comment by MD-AZMAL** [comment]
Yes, key-based encryption is a part of problem statement, so it is supposed to be done
